modules/recommended-configurable-storage-technology.adoc

```diff
@@ -121,8 +121,19 @@ It is not recommended to use file storage for a hosted metrics cluster deploymen
 
 In an {product-title} hosted logging cluster deployment:
 
-* The preferred storage technology is block storage.
-* Object storage is not configurable.
+* Loki Operator:
+** The preferred storage technology is S3 compatible Object storage.
+** Block storage is not configurable.
+
+* Elasticsearch Operator:
+** The preferred storage technology is block storage.
+** Object storage is not configurable.
```

Comment:
```suggestion
** Object storage is not supported.
```

Reply:
I may not be thinking of this correctly, but should the wording be changed (as suggested) to match the table shown here: https://68232--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/optimization/optimizing-storage#specific-application-storage-recommendations?

Reply:
Excellent point, thank you! :)